lld/docs/ReleaseNotes.rst

```diff
@@ -41,9 +41,25 @@ COFF Improvements
 * Added support for ``--time-trace`` and associated ``--time-trace-granularity``.
   This generates a .json profile trace of the linker execution.
 
+* Prefer library paths specified with ``-libpath:`` over the implicitly
+  detected toolchain paths.
+
 MinGW Improvements
 ------------------
 
+* Added support for many LTO and ThinLTO options.
```

Comment:
Dare I ask what options. If it's < 10 maybe just list them?

Reply:
It's a bunch, a bit over 20 options, if counting all the aliases - added in 5c92c9f34a7dba804479acef62c576d1a170ef1f, 0b51e648307cf6c21c463d3e73e51c03aaa8c9e2 and a08506e374f5938e30a9c13b61a697e8c0e12aa3.

The vague "many" in this case is mostly meant to be "every LTO option that the ELF driver supports, that also is supported in the COFF linker at the moment".